docs-user/bunny-2.md

```diff
@@ -0,0 +1,78 @@
+# Case Study
+## Harnessing parallelism
+
+Consider the [previous case study](bunny.md). The amount of work being done was reduced by some performance fixes, allowing for a much faster frame rate. However, there is still a problem with this code. It's not taking advantage of the parallelism of having a worker thread.
```

Comment:
optional: I'd use "obvious" before "problem" => "there is still an obvious problem with this code".